src/main/java/com/epam/aidial/core/data/Publication.java

```diff
@@ -19,6 +19,7 @@ public class Publication {
     /**
      * Target directory prefix to publish; for example: public/ or public/folder/.
      */
+    String name;
```

Comment:
Javadoc description is wrong. It points out to the field name instead of targetFolder

Reply:
fixed